src/Razor/test/testapps/RestoreTestProjects/RestoreTestProjects.csproj

```diff
@@ -4,6 +4,12 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\SimpleMvc11NetFx\SimpleMvc11NetFx.csproj" />
```

Comment:
Umm, this didn't do anything. Also the newly added projects should go here.

Reply:
No because they require packing in order to successfully restore. 

That’s why it happens at runtime as we spoke about on teams.

That’s also why those projects use a custom nuget package cache (and hence go to disk at that time)

Reply:
RIght. I meant all you did was move these itemgroup entries around 😄. Maybe add a comment about why the new project isn't listed here

Reply:
added a comment